App/views/spaces/details.erb

```diff
@@ -0,0 +1,13 @@
+<h2>
+  <%= flash[:notice] %>
+</h2>
+<div>
+    <p><%= @space.name %></p> 
+    <p><%= @space.description %></p> 
+    <p>£<%= @space.price %> per night</p>
+    <form action='/bookings/new' method='post'>
+      <input type='date' name='date_start'>
+      <input type='date' name='date_end'>
+      <input type='submit' value='Book'>
+    </form>
```

Comment:
To-do tomorrow: add call to action: make a booking

Reply:
ok